CodeEditModules/Modules/AppPreferences/src/Sections/GeneralPreferences/GeneralPreferencesView.swift

```diff
@@ -7,9 +7,11 @@
 
 import SwiftUI
 import CodeEditUI
+import Sparkle
 
 /// A view that implements the `General` preference section
 public struct GeneralPreferencesView: View {
+    @StateObject private var updater = ObservableUpdater()
```

Comment:
Unless I'm missing something, I think unless `GeneralPreferencesView` get initialized at the start of your app - the updater won't get initialized and therefore won't automatically check for updates.

Reply:
Good point. I will move it further up and pass the updater down.

Reply:
Done. Good catch!